Source/WebCore/platform/gstreamer/GStreamerQuirks.cpp

```diff
@@ -76,7 +76,7 @@ GStreamerQuirksManager::GStreamerQuirksManager(bool isForTesting, bool loadQuirk
     const char* quirksList = g_getenv("WEBKIT_GST_QUIRKS");
     GST_DEBUG("Attempting to parse requested quirks: %s", GST_STR_NULL(quirksList));
     if (quirksList) {
-        StringView quirks { std::span { quirksList, strlen(quirksList) } };
+        StringView quirks { unsafeMakeSpan(quirksList, strlen(quirksList)) };
```

Comment:
I'm pretty sure this should be ASCII-only, we could use `StringView::fromLatin1()` here.

Reply:
Can call `span(quirksList)`